experimental/saiexperimentaldashacl.h

```diff
@@ -111,42 +111,76 @@ typedef enum _sai_dash_acl_rule_attr_t
     SAI_DASH_ACL_RULE_ATTR_DASH_ACL_GROUP_ID,
 
     /**
-     * @brief List matched key dip
+     * @brief Ternary matched key dst_tag
      *
-     * @type sai_ip_prefix_list_t
+     * @type sai_uint32_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_DASH_ACL_RULE_ATTR_DST_TAG,
+
+    /**
+     * @brief Ternary matched mask dst_tag
+     *
+     * @type sai_uint32_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_DASH_ACL_RULE_ATTR_DST_TAG_MASK,
+
+    /**
+     * @brief Ternary matched key src_tag
+     *
+     * @type sai_uint32_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_DASH_ACL_RULE_ATTR_SRC_TAG,
+
+    /**
+     * @brief Ternary matched mask src_tag
+     *
+     * @type sai_uint32_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_DASH_ACL_RULE_ATTR_SRC_TAG_MASK,
+
+    /**
+     * @brief Optional matched key dip
+     *
+     * @type sai_ip_address_t
```

Comment:
I have submitted a PR (https://github.com/opencomputeproject/SAI/pull/1829) for the TAG feature, this design is reviewed by @marian-pritsak

Reply:
is that different from auto generated ?

Reply:
The TAG feature is still being discussed, so we don't have a final conclusion for it.

Reply:
does it matter at this stage? since it's still experimental, does anyone use it at all at that point? it could be updated to keep headers in sync, and changed then at any point